source/deployment-considerations/colocation.txt

```diff
@@ -0,0 +1,105 @@
+.. _rm-colocation:
+
+==========
+Colocation
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In the context of Relational Migrator, colocation refers to where the 
+servers, networking equipment, and databases are located. This page 
+outlines location specific details to consider when using Relational 
+Migrator.
+
+How to Run Relational Migrator
+------------------------------
+
+The following table outlines some high level recommendations for how to 
+run Relational Migrator. Also note that the 
+:ref:`Kafka deployment model <kafka-intro>` provides greater 
+resilience and scalability and can be used with any of the 
+deployment options.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Use case 
+     - Recommended environment
+
+   * - Smaller scale or debugging and testing jobs that run for 
+       less than four hours. 
+     - Laptop / Desktop / Docker
+
+   * - Larger scale production jobs that run for more than four 
+       hours.
+     - Remote server / EC2 / Azure VM
+
+Where to run Relational Migrator
```

Comment:
```suggestion
Where to Run Relational Migrator
```

Reply:
Woops sorry I missed this from the last round.